github.go

```diff
@@ -2,6 +2,8 @@ package main
 
 import (
 	"fmt"
+	"github.com/iver-wharf/wharf-core/pkg/ginutil"
+	"github.com/iver-wharf/wharf-core/pkg/problem"
```

Comment:
nit: Could you move these to down near the other `github.com/iver-wharf/...` imports?